.github/workflows/release.yml

```
@@ -264,8 +264,9 @@ jobs:
   # AI-powered changelog generation using shared workflow
   generate-changelog:
     name: Generate Changelog
+    continue-on-error: true
```

actionlint (1.7.11)

[error] when a reusable workflow is called with "uses", "continue-on-error" is not available. only following keys are allowed: "name", "uses", "with", "secrets", "needs", "if", and "permissions" in job "generate-changelog"

(syntax-check)